tools/df/docker/bird-fi-test/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# base image 
+ARG BASE_IMAGE=276327358223.dkr.ecr.us-east-1.amazonaws.com/pipedream-dev
+ARG TAG=latest
+ARG BIRD_TAG=2.0.4-4.df
+ARG ARCH=amd64
+
+FROM ${BASE_IMAGE}:${TAG}
+
+ENV BIRD_TAG=${BIRD_TAG}
+ENV ARCH=${ARCH}
+
+WORKDIR /home/support
+ADD media media
+
+# bird2-2.0.4-4.df-amd64.deb
+RUN bash -c "dpkg -i /home/support/media/bird2-2.0.4-4.df-amd64.deb"
```

Comment:
Hardcoding the bird version here doesn't seem intended.

Reply:
We are going to be moving bird2 into a conda package to more easily support Ubuntu18 so we probably don't want to do this